rollbar/__init__.py

```diff
@@ -287,7 +287,8 @@ def _get_pylons_request():
     'locals': {
         'enabled': True,
         'safe_repr': True,
-        'sizes': DEFAULT_LOCALS_SIZES
+        'sizes': DEFAULT_LOCALS_SIZES,
+        'whitelisted_types': []
```

Comment:
Uh, is it `whitelist_types` or `whitelisted_types`? Looks like the variable that `SerializableTransform` takes is the former. Any reason for them to be different?